controllers/v1/library/individualController.js

```diff
@@ -0,0 +1,255 @@
+/**
+ * name : individualController.js
+ * author : Aman
+ * created-date : 04-Jun-2020
+ * Description : Individual library related information.
+ */
+
+const individualHelper = require(MODULES_BASE_PATH + "/library/individual/helper");
```

Comment:
@aman-tunerlabs - Let's call it individualAssessment, just individual doesn't make sense